tests/cases/issues/issue_311/test_issue_311/conftest.py

```diff
@@ -0,0 +1,7 @@
+from pytest_cases import fixture, parametrize_with_cases
+
+
+@fixture(scope='class')
+@parametrize_with_cases('arg', cases='cases', scope='class')
+def class_scoped(arg):
```

Comment:
This fixture does not appear to be used in the test. To be sure that the test works (and not only the pytest initial collection/setup), I would recommend to use this fixture somewhere in the test. In particular, the test does not at all validate that the scope is actually used.

This might require to create a second test, and to wrap both in a class..